app/bundles/EmailBundle/Controller/Api/EmailApiController.php

```diff
@@ -126,6 +126,7 @@ public function sendLeadAction($id, $leadId)
             $post       = $this->request->request->all();
             $tokens     = (!empty($post['tokens'])) ? $post['tokens'] : [];
             $assetsIds  = (!empty($post['assetAttachments'])) ? $post['assetAttachments'] : [];
+            $ignoreDNC  = ($post['ignoreDNC'] === 'true') ? true : false;
```

Comment:
This would generate a warning If param ignoreDNC does not exist, should be

`$post['ignoreDNC'] ?? false`

Reply:
@kuzmany Am I mistaken or does `$post['ignoreDNC'] ?? false` only check for existence? Wouldn't that cause mautic to ignore DNC if `ignoreDNC=false`?

Reply:
It's short syntax of

`isset($post['ignoreDNC']) ? $post['ignoreDNC'] :  false `

Reply:
Can we do this?
`$ignoreDNC  = (($post['ignoreDNC'] ?? false) === 'true') ? true : false;`

I should add I'm not a PHP programmer, and don't know the peculiarities of the language.

Reply:
Should be

            `$ignoreDNC  = InputHelper::boolean($post['ignoreDNC'] ?? false);`